src/dmtest/gendatablocks.py

```diff
@@ -0,0 +1,663 @@
+#!/usr/bin/python3
+
+#
+# %COPYRIGHT%
+#
+# %LICENSE%
+#
+# $Id$
+
+from __future__ import with_statement
```

Comment:
Since dmtest is Python 3 only and the with statement was introduced in 2.x, I don't think this is needed.